### DISCUSSION TITLE
`System.Random` static methods

### DISCUSSION THREAD

**Discussion by FilipToth** [dotnet/runtime#62029] [Ideas] [answered]
There're a lot of cases where we just need to generate one random `Int32`, to do this we need to instantiate the `System.Random` type and call the `System.Random.Next` instance method.  Would it be a good idea to introduce a static method for this common case? 

```c#
namespace System
{
    public class Random
    {
        public static int NextInt32()
    }
}
```
Please note that the method name is temporary,
In most basic cases we don't care about the `seed` argument, but we can make an overload of the proposed static method that takes in a seed argument.

PS: I didn't want to create an API proposal, because I just want some feedback on this idea. And I'm not 100% sure if this is a duplicate.

**Comment by bartonjs** [comment, marked as answer]
That's what `System.Random.Shared` is for:  `System.Random.Shared.Next()`

**Reply by FilipToth** [reply]
Ok, Thank you. I didn't notice that while reading the source.
